atlp-rwanda/rca-barefoot-team6-bn
# Ft register user confirm 184638320

**What does this PR do?**
Register User and Verify registered user

**Description of Task to be completed?**
Implementing user registration and email verification

How should this be manually tested?
Use Postman to create user with this route: ${Basic URL}/api/users
Get that user with ${Basic URL}/api/users to view his token
Paste the token in this route ${Basic URL}/api/users/verify-email/${token}

**Any background context you want to provide?**
Installing libraries needed like bcryptjs
Create app password from your google account to use as APP_PASS env variable
Ensure also that you have EMAIL variable in env file
Then you will be able to send emails to the created users

**What are the relevant pivotal tracker/Trello stories?**
#184638320, #184638371
